components/raftstore/src/store/config.rs

```diff
@@ -327,6 +327,18 @@ pub struct Config {
     /// triggered.
     pub raft_write_size_limit: ReadableSize,
 
+    /// When the size of raft db writebatch is smaller than this value, write
+    /// will yield for a while to make the writebatch larger, which will reduce
+    /// the write amplification.
+    #[doc(hidden)]
+    pub raft_write_batch_size_hint: ReadableSize,
+
+    /// When the size of raft db writebatch is smaller than this value, write
+    /// will yield for a while. This is used to reduce the write amplification.
+    /// Unit: nanoseconds.
+    #[doc(hidden)]
+    pub raft_write_yield_interval: u64,
```

Comment:
`raft_write_wait_dur` is a better name, users don't understand what does yield mean. Please also update the comments:
"When the size of raft write batch is smaller than batch-size-hint value, write will wait for a while for a larger batch ..."